OxfordBodleian/Juel-Jensen/BDLaethf30.xml

```diff
@@ -180,38 +702,228 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <gap reason="lost"/>ድ፡ ወመ<gap reason="lost"/>ኵሎ፡
+                                    <gap reason="lost"/>
+                                   <cb n="b"/> አሜን፡ ወልድኪ፡ ሣ<gap reason="illegible"/>ይክፍለኒ፡ ሰአሊ፡ ለነ፡ ቅድስት። ። ። 
+                                </explicit>
+                            </msItem>
+                    </msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="page">5</measure>
+                                   <!-- leaf dimensions missing
+                                    <dimensions type="folio" unit="mm">
+                                        <height></height>
+                                        <width></width>
+                                    </dimensions>
+                                    <note corresp="#leafdim">Data on leaves dimensions taken from <locus target="#"/>.</note>-->
+                                </extent>
+                           
+                                <condition key="deficient">
+                               Significant parts of the leaves, including text, have been lost on <locus target="#150"/> and <locus target="#152"/>.
+                                </condition>
+                            </supportDesc>
+                            
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="22">
+                                    <locus from="148ra" to="152vb"/>
+                                    <note corresp="#textarea2 #margin2">Data on text area and margin dimensions taken from <locus target="#148ra"/>.</note>
+                           <!--        
+                               measurements missing
+                               <dimensions unit="mm" xml:id="textarea3">
+                                        <height></height>
+                                        <width></width>
+                                    </dimensions>
+                                    <dimensions type="margin" unit="mm" xml:id="margin3">
+                                        <dim type="top"></dim>
+                                        <dim type="bottom"></dim>
+                                        <dim type="right"></dim>
+                                        <dim type="left"></dim>
+                                    </dimensions>-->
+                                    <note> The characters per line per column are 15-17.</note>
+                                    <ab type="pricking">Ruling is visible. Primary pricks are mostly visible, ruling pricks are not visible due to the bad state of condition
```

Comment:
I usually encounter 'bad condition' or 'bad state of preservation' in papers related to conservation, what do you think?

Reply:
I would absolutely follow your judgement, I thought nothing in particular when writing this.